src/cash_protocols.h

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2019 The Bitcoin Unlimited developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_CASH_PROTOCOLS_H
+#define BITCOIN_CASH_PROTOCOLS_H
+
+#include <stdint.h>
+
+/* clang-format off */
+
+enum CASH_PROTOCOLS : uint32_t
+{
```

Comment:
I would add this URL which is the authoritative source for the OP_RETURN prefix 
https://github.com/bitcoincashorg/bitcoincash.org/blob/master/etc/protocols.csv

didn't properly check but it seems like there's something missing from the enum definition.

Reply:
added url, the list in the file is probably incomplete now since i had added it a while ago.